api/src/pcapi/admin/custom_views/beneficiary_user_view.py

```diff
@@ -82,6 +83,34 @@ def beneficiary_deposit_type_formatter(view, context, model, name) -> Markup:
     )
 
 
+def beneficiary_total_amount_formatter(view, context, model, name) -> Markup:
+    amount = get_domains_credit(model)
+    all_spent = "0"
+    if amount is not None and amount.all is not None:
+        all_spent = amount.all.remaining
+    return Markup(f"""<span>{all_spent}</span>""")
```

Comment:
Ton code fonctionne très bien et il n'y a pas de trou de sécurité. Néanmoins, je proposerais plutôt d'utiliser cette forme-là : 

```suggestion
    return Markup("<span>{all_spent}</span>").format(all_spent=all_spent)
```


En effet, je vais (essayer de) proposer l'ajout d'une nouvelle règle pylint qui va vérifier qu'on n'utilise pas la forme `Markup(f"{variable}")`. Par le passé, ça nous a valu des failles XSS. Car ce qu'on passe à `Markup()` directement est sensé être *safe*. Ici, c'est un entier, donc pas de problème. Mais ça pourrait être une chaîne de caractères non contrôlée et là ce serait problématique. La règle pylint permettrait de dire au développeur : attention, il y a peut-être une faille de sécurité ici, mieux vaut utiliser `Markup.format()`. Encore une fois, ici ça n'est pas totalement nécessaire, mais par souci d'homogénéité et pour prendre les "bonnes habitudes", je pense qu'il vaut mieux utiliser `Markup.format()`. :)

(idem dans les autres fonctions)


Si besoin, petite démo du problème avec `Markup()` :

```python
>>> uncontrolled_string = "<script>window.alert('oups !')</script>"
>>> # utilisation directe, bim XSS:
>>> str(markupsafe.Markup(uncontrolled_string))
"<script>window.alert('oups !')</script>"
>>> # utilisation de Markup.format(), hop la chaîne de caractères est "escaped", c'est bon !
>>> str(markupsafe.Markup("{data}").format(data=uncontrolled_string))
'&lt;script&gt;window.alert(&#39;oups !&#39;)&lt;/script&gt;'
```